lib/entities/request/RelayRequest.ts

```diff
@@ -0,0 +1,63 @@
+import { defaultRequestKey, QuoteRequest, QuoteRequestInfo } from '.';
+import { DEFAULT_SLIPPAGE_TOLERANCE, NATIVE_ADDRESS, RoutingType } from '../../constants';
+
+export * from './ClassicRequest';
+export * from './RelayRequest';
+
+export interface RelayConfig {
```

Comment:
shouldnt this mirror routing api more closely? maybe even extend it, since it's basically a wrapper around routing api